sechub-cli/src/mercedes-benz.com/sechub/cli/job.go

```diff
@@ -53,24 +53,44 @@ func waitForSecHubJobDone(context *Context) (status jobStatusResult) {
 			break
 		}
 
-		// PROGRESS bar ... 50 chars with dot, then next line...
-		if newLine {
-			sechubUtil.PrintIfNotSilent(strings.Repeat(" ", 29), context.config.quiet)
-			newLine = false
+		if !context.config.quiet {
+			// Progress dots
+			cursor = printProgressDot(cursor)
 		}
-		sechubUtil.PrintIfNotSilent(".", context.config.quiet)
-		if cursor++; cursor == 50 {
-			sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
-			cursor = 0
-			newLine = true
-		}
-		time.Sleep(time.Duration(context.config.waitNanoseconds))
+
+		time.Sleep(time.Duration(waitNanoseconds))
+		waitNanoseconds = computeNextWaitInterval(waitNanoseconds, context.config.waitNanoseconds)
 	}
 	sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
 
 	return status
 }
 
+// Print progress dots ... 50 dots, then next line...
+func printProgressDot(cursor int) int {
```

Comment:
In case you do not define the size, the underlying CPU architecture/platform will select a size (either 32 bit or 64 bit).

```suggestion
func printProgressDot(cursor uint64) uint64 {
```

Reply:
The value of the `cursor` variable is between 0 and 50.
So `uint8` as type is sufficient.
Changed to `uint8`.